app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
```

Comment:
Using the ERD diagram, there is author_id as a foreign key.
this should be` belongs_to: author, class_name: 'User'`